### PR TITLE
Issue #94: Announce when toggling master track from stereo to mono

### LINKED ISSUE #94: Speak the toggle master mono command
Osara should speak when pressing shift+f9 to toggle master mono, the same way it speaks commands like play, stop and soloed/unsoloed. Currently it speaks nothing.

### PR DESCRIPTION
**Pull request by mulcahy1000**
This should resolve #94

### CODE REVIEW

File: src/reaper_osara.cpp [hunk 1]
```diff
@@ -830,6 +830,10 @@ void postSetSelectionEnd(int command) {
 	fakeFocus = FOCUS_RULER;
 }
 
+void postToggleMasterMono(int command) {
+	outputMessage(GetToggleCommandState(command) ? "master track mono" : "master track stereo");
```
Comment (jcsteh) [inline src/reaper_osara.cpp]: Please change this to "master mono"/"master stereo", as I don't think track is needed to explain what you've pressed and we can thus safely reduce verbosity.

File: src/reaper_osara.cpp [hunk 2]
```diff
@@ -922,6 +926,7 @@ PostCommand POST_COMMANDS[] = {
 	{40696, postRenameTrack}, // Track: Rename last touched track
 	{40175, postToggleItemMute}, // Item properties: Toggle mute
 	{40626, postSetSelectionEnd}, // Time selection: Set end point
+	{ 40917, postToggleMasterMono}, //Master track: Toggle stereo/mono (L+R)
```
Comment (jcsteh) [inline src/reaper_osara.cpp]: nit (for consistency): Remove space after left brace. Add space between "//" and comment.

**Comment by jcsteh** [comment]
I took the liberty of fixing this up myself, since it's been sitting here for a while. :)

**Comment by chrisbelle2015** [comment]
You are a fine fellow 'grin'.

**Pull request merged**